src/Pages/ProductDetail/ProductDetailDesc.js

```diff
@@ -0,0 +1,225 @@
+import React, { useState, useEffect } from "react";
+import Accordion from "./Accordion/Accordion";
+import Philosophy from "./Component/Philosophy";
+import PairProduct from "./PairProduct";
+import Parallax from "react-rellax";
+import theme from "../../styles/themes";
+import styled from "styled-components";
+
+function ProductDetailDesc(
+  name,
+  group,
+  volume,
+  mainColor,
+  colorGroup,
+  price,
+  description,
+  note
+) {
+  const [count, setCount] = useState(0);
+
+  const countAdd = () => {
+    if (count < 10) {
+      setCount(count + 1);
+    }
+  };
+
+  const countSubtract = () => {
+    if (count >= 1) {
+      setCount(count - 1);
+    }
+  };
```

Comment:
```suggestion
  const countAddSubtract = (e) => {
    const isAdd = e.target.dataset.operator === "add"
    const condition = isAdd ? count < 10 : count >= 1
    const num = isAdd ? 1 : -1
    
    if (condition) setCount(count + num)
  };
```
지금 방식처럼 사용하셔도 좋습니다만 이런 방식도 있다는 걸 알려드립니다. 해당 함수가 쓰이는 태그에 `data-operator="add"`처럼 속성 추가해서 사용하시면 됩니다. `onClick={() => func("+")}`처럼 사용할 수도 있는데... 사실 이벤트 리스너 자리에 즉시 실행함수로 함수 인자를 넘겨주는 건 좋은 방법이 아니라서 이 방법을 권장합니다. 이해하기 어려우시면 찾아주세요! 일단 패스하고 싶으시면 그렇게 하셔도 좋습니다.

참고 : https://developer.mozilla.org/ko/docs/Web/API/HTMLElement/dataset

Reply:
wow 참고할게요!! 좋은 정보 감사합니다👍